js/index.js

```diff
@@ -0,0 +1,41 @@
+$('document').ready(function(){
+
+	console.log('ready');
+
+	$('#submit-btn').click(hitSubmit);
+
+
+	function hitSubmit(event) {
+		event.preventDefault();
+
+		var userInput = $('#city-type').val().trim().toLowerCase();
+
+
+			if (userInput ==='nyc'||userInput ==='new york city'
```

Comment:
These `if` `else if` statements should be indented the same as the `var userInput` line above.

```js
function hitSubmit(event) {
  event.preventDefault();
  var userInput = $('#city-type').val().trim().toLowerCase();

  if (userInput ==='nyc'||userInput ==='new york city'||userInput ==='new york') {
    $('body').removeClass('sf la austin sydney').addClass('nyc');
  } else if (userInput ==='san francisco'||userInput ==='sf'||userInput ==='bay area')
    $('body').removeClass('nyc la austin sydney').addClass('sf');
  } // ...
}
```